index.js

```diff
@@ -138,11 +125,25 @@ function createServer({
           waitUntil: 'domcontentloaded'
         });
         await page.waitForTimeout(2000);
+
+        // Get the "viewport" of the page, as reported by the page.
+        const pageDimensions = await page.evaluate(() => {
+          let sum = 0;
+          document.body.childNodes.forEach(el => {
```

Comment:
@maggievu-pdftron can you add some comments explaining why we look at the sum of childNodes of body. I know you mentioned it to me but let's document because it is not obvious from reading the code. You can go into why it gives us a better approximation for the height.

Same for the function in the inject folder.